elephant/signal_processing.py

```diff
@@ -134,6 +135,111 @@ def zscore(signal, inplace=True):
         return result[0]
     else:
         return result
+    
+    
+def cross_correlation_function(signal, ch_pairs, dt=1., env=False, nlags=None):
+
+    """
+    Computes unbiased estimator of the cross-correlation function.
+    
+    Calculates the unbiased estimator of the cross-correlation function 
+    R(tau) = 1/(N-|k|) R'(tau), where R'(tau) = E[x(t)*y(t+tau)] for signal1 
+    and signal2 in a pairwise manner, i.e. signal1[0] vs signal2[0], signal1[1] 
+    vs signal2[1] and so on. The cross-correlation function is obtained 
+    by np.correlate. signal1 and signal2 are zscored beforehand.
+    Alternatively returns the Hilbert envelope of R(tau), which is useful to 
+    determine the correlation length of oscillatory signals.
+
+    Parameters
+    -----------
+    signal : neo.AnalogSignal
+        Signal that contains the LFP channels
+    ch_pairs : list (or array with shape (N,2))
+        list with N channel pairs for which to compute cross-correlation, 
+        each element of list must contain 2 channel indices
+    dt : float
+        Defines sampling period
+    env: bool
+        Return Hilbert envelope of cross-correlation function
+    nlags: int
+        Defines number of lags for cross-corelation function. Floats will be 
+        rounded to nearest integer. Output is of length 2*nlags+1. If None, 
+        len(tau) = Nt
+
+    Returns
```

Comment:
The description of the return values seems to be incomplete. How about going more into details about the envelope / rho calculation. Also mention tau here.